docs/Carcols.md

```diff
@@ -522,3 +529,110 @@ _
    <castShadows value="false"/> --If True = Light will cast shadows of Player/Car
 </Item>
 ```
+
+# Modkits
+
+## Overview
+
+The carcols controlls the modkits items that are on the vehicle. This includes things like aftermarket parts such as exhausts, trims, steering wheels etc. There are a lot of options avaliable to control what the modkits do. 
+
+### Step 1
+
+To keep the file formatted in a way that aligns with GTA, the modkit items should be put at the very beginning of the carcols file. Below the following lines:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<CVehicleModelInfoVarGlobal>
+  ```
+
+### Step 2
+
+On line 3 write:
+  
+```xml 
+<Kits>
+  <Item>
+    
+  </Item>
+</Kits>
+```
+
+This will tell the carcols we are starting the options for the modkits. Once you have this in place, on the line below the opening <Item> tag, we will now define the modkit name. This modkit name will allow the modkit set up that you will create be used anywhere as long as the vehicle & modkits are actively streamed.
+ 
+### Step 3
+  
+```xml
+<kitName>myVehicle_modkit</kitName>
+<id value="47543" />
+<kitType>MKT_SPECIAL</kitType>
+```
+
+The modkit name can be anything. Normally, the name will end in _modkit. 
+
+The ID value below it is very similar to the siren ID value. It uniquely identifies the modkit and makes sure the modkits are unique to the vehicle. 
+  
+For the kitType there isn't a lot of information on what this does, therefore its best to leave it as MKT_SPECIAL. 
+ 
+### Step 4
+  
+Now we shall get into defining the individual modkit parts. Once you have exported the modkit items and have put them in your stream folder we can move on. First, you will need to define you are making visable mods by opening the following tags:
+  
+  
+```xml
+<visibleMods>
+
+</visibleMods>
+```
+  
+This will allow us to create individual items for each individual parts. 
+  
+### Step 5
+  
+Now we can define individual parts:
+```xml
+<Item>
+  <modelName>MODKIT_NAME</modelName>
+  <modShopLabel>MODKIT_SHOP_LABEL</modShopLabel>
+  <linkedModels />
+  <turnOffBones>
+    <Item>misc_*</Item>
+  </turnOffBones>
+  <type>VMT_*</type>
+  <bone>chassis</bone>
+  <collisionBone>chassis</collisionBone>
+  <cameraPos>VMCP_DEFAULT</cameraPos>
+  <audioApply value="1.000000" />
+  <weight value="20" />
+  <turnOffExtra value="false" />
+  <disableBonnetCamera value="false" />
+  <allowBonnetSlide value="true" />
+ </Item>
+ ```
```

Comment:
Would be better to re-structure this. The metadata files are meant to be more objective, information based rather than instructions.

Instructions can go in the tutorial page.